composer.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "jobcloud/php-kafka-schema-registry-client",
+    "license": "MIT",
+    "description": "Kafka Schema Registry API Client",
+    "keywords": ["kafka", "schema", "registry", "php"],
+    "autoload": {
+        "psr-4": {
+            "Jobcloud\\KafkaSchemaRegistryClient\\": "src/",
+            "Jobcloud\\KafkaSchemaRegistryClient\\Tests\\": "tests/"
+        }
+    },
+    "authors": [
+        {
+            "name": "Jobcloud AG",
+            "homepage": "https://www.jobcloud.ch/c/en/about/team/?team=13"
+        }
+    ],
+    "config": {
+        "sort-packages": true
+    },
+    "require": {
+        "php": ">7.3",
+        "ext-json": "*",
+        "pimple/pimple": "^3.2",
+        "symfony/cache": "^5.0",
+        "nyholm/psr7": "^1.2",
+        "kriswallsmith/buzz": "^1.0",
+        "psr/http-message": "^1.0"
+    },
+    "require-dev": {
+        "squizlabs/php_codesniffer": "^3.4.2",
+        "phpunit/phpunit": "^8.3.4",
+        "phpstan/phpstan": "^0.11.8",
+        "rregeer/phpunit-coverage-check": "^0.3.1",
+        "kwn/php-rdkafka-stubs": "^2.0.0",
```

Comment:
that's why ci is failing, sry i missed this, we can kick the stubs out as well ;
```suggestion
```